moon-store-mac-os/moon-store-mac-os/Helpers/Components/MSImagePicker/MSImagePickerViewModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  MSImagePickerViewModel.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 1/2/25.
+//
+
+import Foundation
+import PhotosUI
+import SwiftUI
+
+final class MSImagePickerViewModel: ObservableObject {
+    @Published var selectedItem: PhotosPickerItem?
```

Comment:
Usa esto en la vista ya que no maneja mas informacion como para ser un viewModel `selectedItem: PhotosPickerItem?`. y puedes cambiar el nombre a utility
```suggestion
final class MSImagePickerUtility {
```

Reply:
Debe ser observable ya que estamos utilizando la variable published.

Reply:
por eso dije pasar esa variable al view

Reply:
Listo